sql_generators/glean_usage/templates/metrics_clients_last_seen.query.sql

```diff
@@ -1,3 +1,18 @@
+{{ header }}
+
+{% raw %}
+{% if is_init() %}
+{% endraw %}
+SELECT
+    *
+  FROM
+    `{{ project_id }}.{{ app_name }}.metrics_clients_daily` AS m
+  WHERE
+    FALSE
```

Comment:
We need to get the right metrics fields here:

```suggestion
SELECT
  DATE(@submission_date) AS submission_date,
  _current.client_id,
  _current.sample_id,
  _current.normalized_channel,
  _current.n_metrics_ping,
  _current.days_sent_metrics_ping_bits AS days_sent_metrics_ping_bits,
  {% if app_name in metrics -%}
  {% for metric in metrics[app_name] -%}
    _current.{{metric}} AS {{metric}},
  {% endfor -%}
  {% endif -%}
FROM
  `{{ project_id }}.{{ app_name }}.metrics_clients_daily` AS _current
WHERE FALSE
```